toolbox

```diff
@@ -864,7 +869,7 @@ create()
             --volume "$home_canonical":"$home_canonical":rslave \
             --volume /etc:/run/host/etc \
             --volume /dev:/dev:rslave \
-            --volume /media:/media:rslave \
+            $media_path_bind \
```

Comment:
Is this a typo? :)

Wasn't `/run/media` the missing path? This line is replacing the `/media` entry, not `/run/media`.

Reply:
Doh, you're right.